docker/ssl/authorization.js

```diff
@@ -5,7 +5,7 @@ function authenticate(s) {
     if (!s.variables.ssl_client_s_dn || !s.variables.ssl_client_s_dn.length ||
         !s.variables.ssl_client_verify || s.variables.ssl_client_verify != "SUCCESS") {
         s.deny();
-        return
+        return;
```

Comment:
can you do the same in UI repo

Reply:
Sure. I'll change it in devops also, that's where I spot it

Reply:
[Here](https://github.com/mainflux/ui/pull/105) it is
I couldn't even start nginx without this fix in kubernetes because of this syntax error, but obviously in other environments this is not an issue.